src/gaps/QueryFilterParser.ts

```diff
@@ -863,11 +904,12 @@ export async function executeIntervalELM(
 ): Promise<ParsedFilterInterval | GracefulError> {
   // make sure the interval created based on property usage from the query source
   const propRefInInterval = findPropertyUsage(intervalExpr, undefined);
+  const aliasRefInInterval = findAliasUsage(intervalExpr, undefined);
```

Comment:
This is a good point but I don't want to jump to the conclusion that we need to do a similar lookup for AliasRefs right at the moment. It would require considerable more added testing, perhaps we should save this for later when we have more time.